floorp/browser/locales/en-US/floorp.ftl

```diff
@@ -217,9 +217,9 @@ use-custom-useragent-mode =
 
 ## DMR UI
 download-mgr-UI =
- .label = Enable a simple downloads management interface
+ .label = Enable the simple downloads management interface
```

Comment:
```suggestion
 .label = Enable the SimpleUI Download Manager
```
Better to leave it as it was, because "SimpleUI Download Manager" is more descriptive and specific than the new label.

Reply:
to be honest, I had no idea it was called 'SimpleUI' as the string before my commits similarly called it 'simple downloads management interface'.